opensearch/opensearch.yml

```diff
@@ -3,3 +3,13 @@ http.cors.enabled : true
 http.cors.allow-origin: "*"
 http.cors.allow-methods: OPTIONS,HEAD,GET,POST,PUT,DELETE
 http.cors.allow-headers: X-Requested-With,X-Auth-Token,Content-Type,Content-Length
+plugins.security.ssl.transport.enforce_hostname_verification: false
```

Comment:
If you can comment on the false here as well that would be nice. I don't know if yml has comments.
`# Self-signed certificates so no hostname verification`